libs/designsystem/src/lib/components/modal/alert/alert.component.spec.ts

```diff
@@ -1,150 +1,104 @@
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
-import { By } from '@angular/platform-browser';
 import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
```

Comment:
Why are the changes to this test suite necessary?

Reply:
I have written a short description for this in the commit-message:
"Directly setting the input on the component did not work with OnPush
change detection, so spectator is used in alert test now, while we
were at it. This way we can interact with the component via setInput()."